server/routes/shared_api/observation/point.py

```diff
@@ -69,8 +69,10 @@ def point_within():
   if not variables:
     return 'error: must provide a `variables` field', 400
   date = request.args.get('date') or 'LATEST'
+  facet_id = request.args.get('facetId') or None
```

Comment:
shall we use "facetIds" here as well for consistency?

Reply:
since this is only a single facetId, I think facetId makes more sense? wdyt?

Reply:
Can you remind me what does a list of facet id filter implies? Say i want to fetch [facet1, facet2], but some data has either facet1 or facet2, would that work? 

The result here does return 1 facet, but can the filter be  multiple so I can ask for either 
Census or WorldBank data, whichever has data.